test/spec/vis/vis.spec.js

```diff
@@ -444,43 +446,55 @@ describe("Vis", function() {
 
   describe("dragging option", function() {
 
-    it("should be enabled with zoom overlay and scrollwheel enabled", function() {
+    beforeEach(function() {
       var container = $('<div>').css('height', '200px');
-      var vis = new cdb.vis.Vis({el: container});
-
-      var mapConfig = {
-        updated_at: 'cachebuster',
-        title: "irrelevant",
-        url: "http://cartodb.com",
-        center: [40.044, -101.95],
-        bounding_box_sw: [20, -140],
-        bounding_box_ne: [ 55, -50],
-        zoom: 4,
-        bounds: [[1, 2],[3, 4]],
-        scrollwheel: true,
-        overlays: [
-          {
-            type: "zoom",
-            order: 6,
-            options: {
-              x: 20,
-              y: 20,
-              display: true
-            },
-            template: ""
-          }
-        ],
-      };
+      this.vis = new cdb.vis.Vis({el: container});
+    });
 
-      vis.load(mapConfig);
-      expect(vis.map.get('drag')).toBeTruthy();
+    it("should be enabled with zoom overlay and scrollwheel enabled", function() {
+      var mapConfig = mapConfigCustom(true, true);
```

Comment:
I find `mapConfigCustom(true, true);` a bit hard to understand because of the boolean params.

It would be easier to understand with a single hash param. Something like:

```
var mapConfig = createMapConfig({
  scrollWheel: true,
  zoom: false
});
```